rkt/config/config.go

```diff
@@ -0,0 +1,246 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
That the config will be used only by frontend (the code inside rkt), so it does not need to be generally useful, but only useful for specific needs of a frontend.